openshift/template.yaml

```diff
@@ -79,6 +79,9 @@ parameters:
 - name: INSTALL_RH_CA
   value: "false"
   required: true
+- name: RH_QA_REG_CRED
+  value: ""
+  required: true
```

Comment:
In integration environment we are not using this so why is this required?

Reply:
Additional credentials needed in order to pull container images, that are not part of the regular pull secret.
In our case registry.stage.redhat.io.
Not needed for Brew.
See Jira for additional explainations.